apscheduler/jobstores/mongodb.py

```diff
@@ -1,14 +1,13 @@
 from __future__ import absolute_import
```

Comment:
All `__future__` imports should be removed too.

Reply:
Removed.